src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -470,6 +470,25 @@ namespace
         return std::string();
     }
 
+    int get_medium_priority(Object* object, const TimeValue time)
```

Comment:
This looks good, however we now have four times the same piece of code that looks for a modifier of type `AppleseedObjPropsMod`. That can easily be refactored into a separate function. Want to give this a try? If so, you can do it in a later PR if you prefer.

Reply:
I will try it in another PR